connector_importer/security/security.xml

```diff
@@ -1,9 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <record model="ir.module.category" id="module_category_connector_importer">
+        <field name="name">Importer</field>
+        <field name="sequence">20</field>
+    </record>
+
     <record id="group_importer_user" model="res.groups">
         <field name="name">Connector importer user</field>
         <field name="implied_ids" eval="[(4, ref('base.user_root')), (4, ref('base.group_no_one'))]"/>
```

Comment:
isn't his wrong? `implied_ids` is for groups but is referencing users :thinking:

Reply:
if you check the commit you linked you'll see the difference :wink:

Reply:
Oh yes sorry, I don't know what I was thinking about…